### PR TITLE
[#1716] Pass database to  http://host/di/ 

### PR DESCRIPTION
**Pull request by i5o**

### CODE REVIEW

**Comment by i5o** [comment]
Thanks for testing @lmmrssa, we still need to make "_utils" works.
It basically involves playing with nginx rules, which I guess we are doing on friday

**Comment by lmmrssa** [comment]
@i5o @paulbert  @dogi  we are able to access fauxton with different port tunnel to rpi
Please check pirate.sh:22682 for fauxton and pirate.sh:22680 for planet.
So I think we can merge this PR.

**Comment by i5o** [comment]
Are we going to merge it @lmmrssa ?

Perhaps changing /di to /db before?

File: docker/planet/default.conf [hunk 1]
```diff
@@ -57,7 +57,7 @@ server {
     add_header Content-Type text/plain;
   }
 
-  location /di/ {
+  location /db/ {
```
Comment (i5o) [inline docker/planet/default.conf]: :cry: :disappointed:

**Comment by paulbert** [comment]
Almost merged this before realizing that the changes to `environment.ts` would break our development environment.  Only needed to change the value for production where we have docker/nginx.

**Pull request merged**